atmo/forms/mixins.py

```diff
@@ -75,7 +75,8 @@ def __init__(self, *args, **kwargs):
         self.required_filefields = []
 
         field_order = []
-        for name, field in self.fields.items():
+        fields = self.fields.copy()
```

Comment:
Since it was complaining that OrderedDict is being modified during iteration. That has something to do with dict subclasses in py2k return views, and not the dicts themselves, so it raises that error to make sure the state isn't modified during iteration.